client/post-editor/editor-revisions-list/header.jsx

```diff
@@ -7,15 +7,48 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
 
-const EditorRevisionsListHeader = ( { numRevisions, translate } ) => {
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+const EditorRevisionsListHeader = ( {
+	numRevisions,
+	translate,
+	selectNextRevision,
+	selectPreviousRevision,
+} ) => {
 	return (
-		<div className="editor-revisions-list__header">
-			{ !! numRevisions &&
-				translate( '%(revisions)d revision', '%(revisions)d revisions', {
-					count: numRevisions,
-					args: { revisions: numRevisions },
-				} ) }
+		<div>
+			<div className="editor-revisions-list__header">
+				{ !! numRevisions &&
+					translate( '%(revisions)d revision', '%(revisions)d revisions', {
+						count: numRevisions,
+						args: { revisions: numRevisions },
+					} ) }
+			</div>
+			<div className="editor-revisions-list__navigation">
```

Comment:
Can / should we use a [`ButtonGroup`](https://wpcalypso.wordpress.com/devdocs/design/button-group) here instead of a div?

[README](https://github.com/Automattic/wp-calypso/blob/0d4e10481892accae3e1d30c4cda5ded2d3ba66f/client/components/button-group/README.md).

Reply:
Positioning the button container is more straightforward when it's a block element, and I was dubious about setting `display: block` on a `span` when I could just use a `div`. But I can see a `ButtonGroup` is better semantically.